### ISSUE TITLE
Proposal to change definition of err:XD0062

### ISSUE THREAD

**Issue by ndw** [xproc/3.0-specification#425]
The current definition of `err:XD0062` is:

````
It is a dynamic error (err:XD0062) if content-type is specified and document-properties has a key "content-type".
````
I propose instead that it should be:

````
It is a dynamic error (err:XD0062) if content-type is specified and document-properties has a key "content-type" with a value that differs from the specified content-type.
````

**Comment by xml-project** [comment]
I would agree to this, but as consensus on #385 you recorded:

> Per 11 June f2f, it's only an error if the content-type attribute is specified and the document-properties map includes a content-type key.

**Comment by ndw** [comment]
Thanks. FWIW, I slipped this into #422.

**Comment by ndw** [comment]
Closed by #422
